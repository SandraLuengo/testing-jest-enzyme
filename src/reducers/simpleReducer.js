const initialState = {
  contact: '',
};

export default (state = initialState, action) => {
  switch (action.type) {
    case 'SIMPLE_ACTION':
      return {
        contact: action.newValue,
      };
    default:
      return state;
  }
};
