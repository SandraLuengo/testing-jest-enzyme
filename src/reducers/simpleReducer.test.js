import simpleReducer from './simpleReducer';


describe('INITIAL_STATE', () => {
  test('is correct', () => {
    const action = { type: 'SIMPLE_ACTION' };
    const initialState = { contact: 1 };
    /* expect(simpleReducer(initialState, initialState)).toEqual(initialState); */
  });
});


describe('SELECT_AVATAR', () => {
  test('returns the correct state', () => {
    const action = { type: 'SIMPLE_ACTION', contact: 1 };
    expect(simpleReducer('', action)).toMatchSnapshot();
  });
});

