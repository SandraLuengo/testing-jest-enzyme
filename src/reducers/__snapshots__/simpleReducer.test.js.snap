// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SIMPLE_ACTION initialState is correct 1`] = `
Object {
  "contact": 1,
  "toggleColor": false,
}
`;
